ci/pipelines/concourse-pypi-resource-pipeline-template.yml

```diff
@@ -188,6 +188,23 @@ jobs:
     params:
       glob: 'dist/*'
 
+- name: concourse-pypi-resource-integration-test-get
```

Comment:
Note for other reviewers on why this is required:
- before we counted on implicit get to do this test for us... and until https://github.com/Pix4D/concourse-pypi-resource/pull/91 it worked fine... what we called before `pypi-acceptance-test` was testing both put and get steps... after I disabled the implicit get, I basically removed half of the tests...

The approach proposed here adds back the get step tests but in a separate job to conform to OPA rules (no implicit get). This is also clearer and the same approach as in: https://builder.ci.pix4d.com/teams/developers/pipelines/s3-resource-simple-master

Reply:
Can we add a comment somewhere mentioning this rationale?